src/routes/user-event.ts

```diff
@@ -0,0 +1,100 @@
+import asyncHandler from 'express-async-handler';
+import httpErrors from 'http-errors';
+import { Router } from 'express';
+import { getUserFromRequest } from '../lib/AuthHelper';
+import { addToCollection, DB_PATHS, updateDocument } from '../lib/DBHelper';
+import { sanitizeString } from '../lib/DataValidator';
+import { getDb } from '../lib/Firebase';
+import { EVENT_STATUS } from '../lib/EventHelper';
+
+const router = Router();
+
+/**
+ * Update event users
+ * Sample JSON POST
+ {
+    "eid": "8UhyXDgfoBAJBboJkMQQ",
+    "uid": "42fdsGW3teCAwwoJkMQQ"
+    "status": (0, 1, 2) 0 = Yes, 1 = No, 2 = Maybe
+ }
+ */
+router.post(
+  '/',
+  asyncHandler(async (req, res, next) => {
+    const user = await getUserFromRequest(req);
+    const eid = sanitizeString(req.body.eid);
+    const status = sanitizeString(req.body.status);
```

Comment:
Don't parse this as a string, it will be a number. Verify that this is a number using `isNumber` function. and check whether it is one of `USER_EVENT_STATUS.YES` or `USER_EVENT_STATUS.MAYBE` or `USER_EVENT_STATUS.NO`